packages/jwt-middleware/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Assert = require('assert');
+const ExpressJwt = require('express-jwt');
+const jws = require('jws');
```

Comment:
I don't think we should be using raw `jws` here. There are a number of things that `jsonwebtoken` provides as a wrapper around `jws` that are important. The biggest one that comes to mind is expiry. The middleware, as it stands would authenticate and authorize requests having expired tokens (`exp` claim).

Additionally, there are probably a bunch of unknown, unknown mistakes we could be making that are avoidable by leveraging the accumulated wisdom in the higher-level `jsonwebtoken` library of ours. `jsonwebtoken`, IIRC, can do the issuer and audience checks automatically as well as checking things like expiry.